src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -735,6 +735,12 @@ private IEnumerable<(string Prefix, long Value, long Index)> InternalGetIdentiti
                         continue;
                     }
 
+                    if (_buildVersionType == BuildVersionType.V3)
```

Comment:
Can we move this to DatabaseSmuggler? We modify documents there based on version type.